src/ralph/accounts/admin.py

```diff
@@ -120,18 +120,25 @@ def user_licence(self, item):
         else:
             return []
 
+    def buyout_ticket(self, item):
+        get_params = {"inventory_number": item.barcode,
+                      "serial_number": item.sn,
+                      "model": item.model,
+                      "comment": item.buyout_date
+                      }
+        url = "?".join(
```

Comment:
wouldn't it be better to use `string.format` to create this url?

Reply:
with @ar4s we decide, that this form look better, but I can change it. Should I?